src/org/zaproxy/zap/extension/websocket/resources/Messages.properties

```diff
@@ -47,14 +47,29 @@ websocket.filter.label.regex.inverse            = Inverse
 websocket.fuzz.fail                             = Error
 websocket.fuzz.success                          = Successful
 websocket.manual_send.fail                      = Unable to send crafted message!
+websocket.manual_send.fail.unable_reopen        = It is not possible to re-open the connection.
```

Comment:
- `is` → `was` and `... the connection because of an error.` (to indicate why it was not possible).
- In some places it's being used `re-open` in others `reopen`, this should be normalised, probably with the latter?